styles.css

```diff
@@ -337,3 +327,1067 @@ body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+
+  /* identical to box height, or 133% */
+
+  letter-spacing: 0.03em;
+
+  /* Primary/B400 */
+
+  color: #6070ff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  flex-grow: 0;
+  margin: 0 0;
+}
+
+.tag-js {
+  /* Tag */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  position: static;
+  width: 73px;
+  height: 24px;
+  left: 114px;
+  top: 0;
+
+  /* Primary/B50 */
+
+  background: #ebebff;
+  border-radius: 8px;
+
+  /* Inside auto layout */
+
+  flex: none;
+  flex-grow: 0;
+  margin: 0 8px;
+}
+
+.tag-js > span {
+  /* javaScript */
+
+  position: static;
+  width: 65px;
+  height: 16px;
+  left: 12px;
+  top: 4px;
+
+  /* Button / Small */
+
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+
+  /* identical to box height, or 133% */
+
+  letter-spacing: 0.03em;
+
+  /* Primary/B400 */
+
+  color: #6070ff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  flex-grow: 0;
+  margin: 0 0;
+}
+
+.card-works-2 {
+  /* Card Works */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  position: static;
+  width: 327px;
+  height: 596px;
+  left: 0;
+  top: 754px;
+
+  /* Neutrals/N0 */
+
+  background: #fff;
+
+  /* Neutrals/N40 */
+
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+
+  flex: none;
+  flex-grow: 0;
+  margin: 88px 0;
+}
+
+.card-works-3 {
+  /* Card Works */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  position: static;
+  width: 327px;
+  height: 596px;
+  left: 0;
+  top: 1438px;
+
+  /* Neutrals/N0 */
+
+  background: #fff;
+
+  /* Neutrals/N40 */
+
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+
+  flex: none;
+  flex-grow: 0;
+  margin: 88px 0;
+}
+
+.card-works-4 {
+  /* Card Works */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  position: static;
+  width: 327px;
+  height: 596px;
+  left: 0;
+  top: 2078px;
+
+  /* Neutrals/N0 */
+
+  background: #fff;
+
+  /* Neutrals/N40 */
+
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+
+  flex: none;
+  flex-grow: 0;
+  margin: 88px 0;
+}
+
+.action {
+  /* Action */
+
+  position: static;
+  width: 295px;
+  height: 72px;
+  left: 0;
+  top: 216px;
+
+  /* Inside auto layout */
+
+  flex: none;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.see-project > button {
+  /* See Project */
+
+  position: static;
+  width: 124px;
+  height: 48px;
+  top: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
```

Comment:
- [ ] The` border-color` and `background-color` on your buttons don't correspond to the Figma design of your chosen template. Please make them `#7f8cff` and `#fff` respectively to match the design 😎 
### Your Button Design
![image](https://user-images.githubusercontent.com/46839548/185626890-42d637e6-3ed5-4c7f-af1f-1e8c37799c46.png)
### Figma Button Design
![image](https://user-images.githubusercontent.com/46839548/185627009-fb3cd4c0-4f4e-477e-ab30-176c3ffc5ac4.png)

- [ ] As a requirement, your buttons should follow the FIGMA design guidelines. Kindly implement the `hover` and `pressed` states for your buttons as shown below 👇 
![image](https://user-images.githubusercontent.com/46839548/185625821-0eee64db-3544-451e-be53-909294f172b3.png)